### PR TITLE
Profile view

### PR DESCRIPTION
**Pull request by MaksZys**

### CODE REVIEW

**Comment by MaksZys** [comment]
## Tasks included in this pull request
- [x] #10 
- [x] #22 
- [x] #26 

moreover
I implemented simple IdentifyService that recognize and return any object by it's id

File: src/models/UserModel.js [hunk 1]
```diff
@@ -25,23 +25,16 @@ export default class UserModel {
...
       }
 
-      addDevice(device) {
-        this.devices.push(device)
-      }
-      
-      removeDevice(deviceId) {
-        //skasowac dany element z danym ID z tablicy
-        remove(this.devices, item => item.id === deviceId);
+      @computed get getFullName(){
```
Comment (kamil7x) [inline src/models/UserModel.js]: It doesn't need to start with `get`. Just `fullName` will be ok

File: src/modules/addDevice/addDevice.view.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+import { Flex, InputItem, List, Button, WhiteSpace, WingBlank, TextareaItem, NoticeBar } from 'antd-mobile';
+import { observable, reaction, observe } from 'mobx';
+import { inject, observer } from 'mobx-react';
+import { Bind } from 'lodash-decorators';
+import DeviceService from '../../services/DeviceService';
+
+@inject('userStore')
+@observer
+class AddDevice extends Component {
+  @observable name = '';
```
Comment (kamil7x) [inline src/modules/addDevice/addDevice.view.js:11]: You can use one observable object eg `@observable device = new DeviceModel()`, so you don't have to do magic in line 48. That's what models are for :)

File: src/modules/addPlace/addPlace.view.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { Flex, InputItem, List, Button, WhiteSpace, WingBlank, TextareaItem, NoticeBar } from 'antd-mobile';
+import { observable, reaction, observe } from 'mobx';
+import { inject, observer } from 'mobx-react';
+import { Bind } from 'lodash-decorators';
+import DeviceService from '../../services/DeviceService';
+import { Select } from 'antd';
```
Comment (kamil7x) [inline src/modules/addPlace/addPlace.view.js]: Try to not mix external imports with internal imports.

For readability, external imports should be first.

File: src/modules/addPlace/addPlace.view.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  @Bind
+  sendToDatabase() {
+    if (this.name.length != 0 &&
+      this.selectedPlaceId != 0) {
+      const data = {
+        name: this.name,
+        officeId: this.selectedPlaceId
+      }
+      PlaceService.postNewPlace(data);
+      this.errorMsg = 'New place added';
```
Comment (kamil7x) [inline src/modules/addPlace/addPlace.view.js]: Wow, that's not an error :D maybe we can use better name?

Comment (kamil7x) [reply src/modules/addPlace/addPlace.view.js]: Or even better, if place is added successfully, redirect user to places list, or this place details view

File: src/routers/devices.router.js [hunk 5]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
+import AddDevice from '../modules/addDevice/addDevice.view';
 
 const DevicesRouter = ({ match }) => (
   <Switch>
     <Route exact path='/devices' render={() => <div>devices</div>} />
-    <Route path="/devices/add_device" render={() => <div>devices add device</div>} />
+    {/* <Route path="/devices/add_device" render={() => <div>devices add device</div>} /> */}
```
Comment (kamil7x) [inline src/routers/devices.router.js]: You can remove commented code

File: src/services/DeviceService.js [hunk 6]
```diff
@@ -12,6 +11,28 @@ class DeviceService {
...
+        'ContentType': 'application/json'
+      },
+    })
+    .catch((err) => {
+      console.log(err);  
+    })
+  }
+
+  deleteDevice(deviceId) {
+    client.delete(`/api/devices/${deviceId}`)
```
Comment (kamil7x) [inline src/services/DeviceService.js:31]: Nice! That's what template strings are for! :)

File: src/stores/UsersStore.js [hunk 7]
```diff
@@ -18,6 +15,22 @@ class UserStore {
...
+
+  setToUsersList(data){
+    this.usersList.push(data);
+  }
+
+  deleteUsersList() {
+    this.usersList = [];
+  }
+
+  @computed get getFullName() {
```
Comment (kamil7x) [inline src/stores/UsersStore.js]: You can remove this and use `UserStore.currentUser.fullName`. If you still want to have this in store, you can `return this.currentUser.fullName` instead

**Review by kamil7x — CHANGES_REQUESTED** [review]

File: src/modules/addPlace/addPlace.view.js [hunk 8]
```diff
@@ -53,7 +53,7 @@ class AddPlace extends Component {
         <WhiteSpace size='xl'/>
         <Flex>
           <Flex.Item>
-            {this.errorMsg && <NoticeBar mode='closable' onClick={this.resetErrMessage}>{this.errorMsg}</NoticeBar>}
+            {this.notificationMsg && <NoticeBar mode='closable' onClick={this.resetErrMessage}>{this.notificationMsg}</NoticeBar>}
```
Comment (kamil7x) [inline src/modules/addPlace/addPlace.view.js]: You forgot to change onClick function

**Review by kamil7x — CHANGES_REQUESTED** [review]

**Review by kamil7x — APPROVED** [review]

**Pull request merged**